onboarding/src/Components/FormsEdit/FormSections.js

```diff
@@ -7,7 +7,18 @@ import FormChoiceEdit from "./SingleChoiceForm/FormChoiceEdit";
 import FormMultiChoiceEdit from "./MultiChoiceForm/FormMultiChoiceEdit";
 import FormAddSection from "./FormAddSection";
 
-function FormSections({ sections, setSections }) {
+function FormSections() {
+  const [sections, setSections] = useState([]);
+  const refValue = useRef(sections);
+  useEffect(() => {
```

Comment:
to to  jest od tego problemu z zapamiętywaniem stanu?  żeby nie znikały następne pola?

Reply:
Tak, w połączeniu z tym ;) 
![image](https://user-images.githubusercontent.com/51723246/96864461-bc1b6b80-1468-11eb-82b9-1a6338d1e1d0.png)